### ISSUE TITLE
How much is the final loss of the PS-FCN network ? 

### ISSUE THREAD

**Issue by bboymoonchild** [guanyingc/PS-FCN#6]
Hi. Now I almost completed implementing your networks to the keras environment.

Now I want to know whether I made the network right, since my network outputs loss value larger than 0.18, when adjusting no scaling/random crop/noise aug. with sculpture dataset.

Can you tell me how much the loss of training should be ? for example, 0.0001 theoretically.

**Comment by guanyingc** [comment]
Hi, thank you for your interests. I have tried checked the old log files. If I have found the correct log file, the final average loss for the last epoch is 0.208. This result includes all data augmentation shown in the code, with both sculpture and blobby datasets. You may implement the dataloader for DiLiGenT dataset and see the results on it.
